hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/upgrade/SCMUpgradeFinalizer.java

```diff
@@ -48,22 +51,15 @@ public SCMUpgradeFinalizer(HDDSLayoutVersionManager versionManager) {
   public StatusAndMessages finalize(String upgradeClientID,
                                     StorageContainerManager scm)
       throws IOException {
-    storageContainerManager = scm;
-    StatusAndMessages response = preFinalize(upgradeClientID, scm);
-    if (response.status() != FINALIZATION_REQUIRED) {
-      return response;
-    }
-    try {
-      getFinalizationExecutor().execute(scm.getScmStorageConfig(), this);
-    } catch (Exception e) {
-      e.printStackTrace();
-      throw new IOException(e.getMessage());
-    }
-    return STARTING_MSG;
+    return super.finalize(upgradeClientID, scm);
```

Comment:
after refactoring we don't need this method as it is just delegating to super.

Reply:
Fixed